src/Services/SmartProxy.js

```diff
@@ -30,6 +31,7 @@ export const SmartProxyApi = createApi({
           data: response.data.filter((element) => element.cluster_id !== ''),
         };
       },
+      keepUnusedDataFor: 5,
```

Comment:
Does it make sense to set this value to 0 instead? So that we **always** do re-fetch when the table is mounted again. 

This could reduce potential cludges in the integration tests for @ikerreyes, and we would not need to wait for 5 seconds to have data updated.